packages/paste-color-contrast-utils/__fixtures__/tokensWithExtraPairings.raw.json

```diff
@@ -163,6 +163,16 @@
       "comment": "Dark shade of destructive link text to be used in interactions",
       "originalValue": "{!palette-red-70}",
       "name": "color-text-link-destructive-dark"
+    },
+    "color-data-visualization-1": {
```

Comment:
I'm not super familiar with the color contrast tools, but why do we only need to test one data viz color?

Reply:
We're testing all the color contrasts in the other test file. This particular fixture is for a unit test to make sure the function that pulls out tokens that need dataviz color contrast checking works correctly.